tt_eager/tt_dnn/op_library/eltwise_binary/eltwise_binary_op.cpp

```diff
@@ -37,13 +36,27 @@ std::map<string, string> get_defines(BinaryOpType op_type, const std::optional<s
             op_name = "mul_tiles";
             op_code = "2";
             break;
-        case BinaryOpType::GT: defines.merge(eltwise_unary_op_utils::get_defines(UnaryOpType::GTZ, std::nullopt, "0", idst)); break;
```

Comment:
not to nitpick on style, but the style before the clang format is so much easier to read

Reply:
I think we should update .clang-format then